### PR TITLE
Fawry CardToken APIs

### PR DESCRIPTION
**Pull request by AhmedAliIbrahim**
This PR enables using Fawry List, create and delete Card Token

### CODE REVIEW

**Comment by amrrbakry** [comment]
Hi @AhmedAliIbrahim,

thank you so much for this valuable contribution 💯 👍 Could you please take a look at the rubocop offenses in circleci and fix them? You can disable the offenses around the `build_request` method as I think it doesn't make much sense to change it. 

Also, could you add your change to the README.md and README-ar.md? Thanks!

I will review again ASAP.

**Comment by AhmedAliIbrahim** [comment]
Hello @amrrbakry,
I'll update the README file ASAP

File: lib/fawry/requests/charge_request.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@
 module Fawry
   module Requests
     module ChargeRequest
-      DEFAULTS = { payment_method: 'PAYATFAWRY', currency_code: 'EGP' }.freeze
+      DEFAULTS = { payment_method: 'CARD', currency_code: 'EGP' }.freeze
```
Comment (amrrbakry) [inline lib/fawry/requests/charge_request.rb:8]: I don't think this should be `CARD` by default, but you can provide it as param for card payments

**Review by amrrbakry — CHANGES_REQUESTED** [review]
LGTM. Please add the unit tests and fix the rubocop offenses (ignore the ones around the `build_request` method), so I can merge. If you need any help, please let me know

**Comment by AhmedAliIbrahim** [comment]
Hi @amrrbakry, I've updated Specs & README

**Review by amrrbakry — APPROVED** [review]

**Comment by amrrbakry** [comment]
@AhmedAliIbrahim Thank you so much for this great contribution! 🎉 👍

**Comment by amrrbakry** [comment]
published as Fawry `v.1.1.1` 🚀 Thanks for contributing 👍

**Comment by AhmedAliIbrahim** [comment]
You're welcome 👍

**Pull request merged**